peps/pep-0740.rst

```diff
@@ -196,6 +196,8 @@ Index changes
 Simple Index
 ^^^^^^^^^^^^
 
+The following changes are made to the :pep:`503` Simple Index:
```

Comment:
The general idea is that PEPs become specs at https://packaging.python.org/en/latest/specifications/ and the PEP itself is then a historical document, and the living specs themselves are updated.

I'm not sure if that has happened yet for PEPs [503](https://peps.python.org/pep-0503/) and [691](https://peps.python.org/pep-0691/)?

They're lacking the banner that others such as [685](https://peps.python.org/pep-0685/) have. (It could also be the specs exist, but we've just not added the banner yet.)

Reply:
> They're lacking the banner that others such as [685](https://peps.python.org/pep-0685/) have. (It could also be the specs exist, but we've just not added the banner yet.)

Yeah, I was wondering about that 😅 -- I see [Simple repository API](https://packaging.python.org/en/latest/specifications/simple-repository-api/) as a specification page in the PyPA specs, but that page looks like its own rewrite of a merger of both PEP 503 and 691.

Reply:
I've replaced these PEP refs with their equivalent packaging refs 🙂